package.json

```diff
@@ -39,15 +39,15 @@
   },
   "devDependencies": {
     "@types/debug": "^4.1.5",
-    "@types/jest": "^24.0.18",
-    "concurrently": "^4.1.2",
-    "graphql": "^14.0.2",
-    "jest": "^24.9.0",
+    "@types/jest": "^26.0.22",
+    "concurrently": "^6.0.2",
+    "graphql": "14.7.0",
+    "jest": "^26.6.3",
     "nodemon": "^1.19.1",
-    "postgraphile": "^4.9.0",
-    "postgraphile-core": "^4.9.0",
+    "postgraphile": "4.9.0",
+    "postgraphile-core": "4.9.0",
```

Comment:
Do we need to pin this exact version?

Reply:
I'm in the habit of pinning them because minor version updates often break the tests due to small changes in the schema. But it's no big deal.. reverted. Had to add `graphql` to `resolutions` because it pulled in an additional v15 package which.. broke the tests :wink:.